WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/FormattableAmountTextFieldViewModel.swift

```diff
@@ -24,7 +24,7 @@ final class FormattableAmountTextFieldViewModel: ObservableObject {
     }
 
     /// When true, the amount will be reset with the new input instead of appending
-    /// 
+    ///
```

Comment:
Note for reviewer: there are a few diffs in this PR to remove trailing space from the git precommit hook checks from a separate PR (not sure why it wasn't checked for everyone)